### PR TITLE
chore: bump to 0.8.0rc2

### PR DESCRIPTION
**Pull request by henryiii**
Includes #112.

### CODE REVIEW

**Comment by henryiii** [comment]
Could I get at least one other maintainer signoff? (@FFY00, @rgommers, @frostming, @pradyunsg)?

I expect to make the full release shortly after if there are no issues with this. I'll adopt it into https://github.com/scikit-build/scikit-build-core/pull/703.

**Review by frostming — APPROVED** [review]

**Review by rgommers — APPROVED** [review]
LGTM. Thanks for the hard work on this release @henryiii

**Pull request merged**
